distribution/docs/src/main/jdocs/content/_tables/org.codice.ddf.spatial.geocoding.plugin.GeoCoderPlugin-table-contents.adoc

```diff
@@ -0,0 +1,22 @@
+:type: documentation
+:status: published
+
+.[[org.codice.ddf.spatial.geocoding.plugin.GeoCoderPlugin]]GeoCoder Plugin
+[cols="1,1,1,4a,1,1," options="header"]
+|===
+
+|Title
+|Property
+|Type
+|Description
+|Default Value
+|Required
+
+|Radius
+|`radiusInKm`
```

Comment:
✏️ We should be consistent in these tables and either use back-ticks for the property name or not.